From 6d78c4ab11d6d9ebcb9ca23823184f91ee998889 Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Mon, 5 Sep 2022 11:52:20 +0200
Subject: [PATCH 06/21] battery: Quiet debug message

This might end up spamming users. Best leave those messages to folks who
are interested in root-causing those problems.
---
 src/up-device-battery.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/up-device-battery.c b/src/up-device-battery.c
index 5874901..734b6a8 100644
--- a/src/up-device-battery.c
+++ b/src/up-device-battery.c
@@ -169,7 +169,7 @@ up_device_battery_estimate_power (UpDeviceBattery *self, UpBatteryValues *cur)
 
 	/* This hopefully gives us sane values, but lets print a message if not. */
 	if (energy_rate < 0.1 || energy_rate > 300) {
-		g_message ("The estimated %scharge rate is %fW, which is not realistic",
+		g_debug ("The estimated %scharge rate is %fW, which is not realistic",
 			   cur->state == UP_DEVICE_STATE_DISCHARGING ? "dis" : "",
 			   energy_rate);
 		energy_rate = 0;
-- 
2.39.1

